spec/system/admin/reports_spec.rb

```diff
@@ -272,11 +284,11 @@
       product1.taxons = [taxon]
       product2.taxons = [taxon]
       variant1.on_hand = 10
-      variant1.update_column(:sku, "sku1")
+      variant1.update_attribute(:sku, "sku1")
```

Comment:
Regarding the Rubocop violation:
```suggestion
      variant1.update!(sku: "sku1")
```

The update method calls validations and the bang version makes sure that the test case fails if a validation fails for an unforseen reason.

While it's good to call validations here, I think, we can't do the same in production code without other changes. In a lot of cases we avoid the validations intentionally. The real solution can be a lot of work and involves calling fewer validations by default and run additional validations when needed.